### PR TITLE
Added SIWFT API Utilities

### PR DESCRIPTION
**Pull request by akama-aka**
# Description

Hello,
I have been working on a module for the last few days to simplify and promote the implementation of the SIWFT API. 

# Features

* Create Transactions
* Get Wallet

# Upcoming
Please read /modules/siwft-utils/readme.md

*Dont push yet!*

### CODE REVIEW

**Comment by SCDerox** [comment]
Hi, thanks for submitting a pull request. When looking at your submission, the following issues were identified:

- This module does not support any user-facing features. CustomBot v3 does not support auxiliary modules. Consider adding features to your modules so that users can directly interact with your module, rather than just allowing API integrations.
- The module is communicating with a third-party API. This API is not well known and lacks legally required documents, such as a complete imprint (contact details are missing, but required as per § 5 TMG), a complete privacy notice (including clear naming of the legal operator of the service), Terms of Service (server returns 404 on the main website, we were able to find the document in your docs) and more. After required documents documentation has been made available, a review from our legal team is required before we can proceed with offering such a module on SCNX, in addition to reviewing whether integrating into an API makes sense from a product perspective. These processes are in place to protect our customers. Additionally, a contract might be required to ensure that our interests and customer data are adequately protected.
- The name of the module likely infringes on a trademark registration. Consider using a name that is not protected by international trademarks. Making a spelling mistake in an international trademark is likely not enough to avoid trademark issues. Additionally, this type of naming creates confusion for customers as they associate the name SWIFT to real-world transaction. Consider rebranding your project.
- The project ("SIWFT") is lacking clear documentation on its website, and before we can consider adding support for such a system, we'd need to be able to get a clear overview of its benefits, functionalities and risks. We do not want to support crypto or crypto-like transactions, where the value of virtual money is tied to real currency or actual work. The central element of the system is concering, with money "being printed" as welcome bonues and similar features

We appreciate your contribution, but please review our comments, and please provide us with additional information where needed. 

I know that this type of response is disappointing, but please understand that we need to make sure that we protect our customers and create the best possible experience for them.

**Comment by SCDerox** [comment]
Also if you are working on this module and don't want it pushed yet, please concert it to a draft, so it's easily distinguishable ^^

**Comment by akama-aka** [comment]
Thank you for your review.

1. A user-interactable feature is planned for future updates.
2. Our imprint includes all information required by Austrian MedienG law. As we don't have a legal operator, we are not required to mention the person again. I will address the issue with the Terms of Service as soon as possible.
3. We plan to rename our API, but we are currently facing challenges in finding a suitable name.
4. I understand the concern about not supporting cryptocurrencies. I also want to clarify that SIWFT is not a cryptocurrency, as it does not use blockchain technology or similar systems. The primary purpose of SIWFT is to create a centralized economy system that is not based on real money or the trading of physical products. I am committed to thoroughly investigating any instances of money laundering or other illegal activities that violate our guidelines. Additionally, we plan to contact the British Money Laundering Authority to better handle such cases.

TL;DR: Real money and physical product trading are not allowed. We plan to contact the British Money Laundering Authority. SIWFT is not intended to be

**Comment by SCDerox** [comment]
Thanks for getting back to us. 

European privacy regulation requires the clear disclosure of who is processing personal data, including the full legal name, address and contact details. This is missing.
Quick googling also showed that you likely need to have additional disclosure in your imprint, as your service can likely be considered indirectly commercial; regardless, our own policies and rules might require a legal notice that is sufficient.
To clarify: I'm not a lawyer (and don't really want to be involved in a legal debate on this matter) and I don't get to make the final decision about whether your service meets our standards, but I advise you to be transparent about your platform's privacy to avoid complications later when the review gets conducted by legal. If your review is denied, we'll be unable to offer your module on SCNX.

Leaving the boring legal stuff behind, I suggest thinking about renaming your project before release, you can ask on our Discord for name suggestions or google "Brand name generator" to look at some ideas (:

Regarding your last point, there was a small misunderstanding: We are not talking about physical products or similar; we are talking about the usage of the currency. If your currency behaves like a "real" world currency, meaning that it can be exchanged for any kind of goods or services, regardless of their form, we won't be able to merge your pull request due to regulatory concerns. If your currency is just used as "play money," meaning it can only be used on Discord or similar platforms without any real-world implications, we won't have any issues with this. However, if your currency offers real-world exchanges or similar activities, we won't be able to provide a platform for it. 


Thanks so much for your patience with this. I apologize for these complications, our goal is to ensure the well-being of our customers.
I greatly appreciate you taking time to make sure we're on the same page.
